content/en/docs/instrumentation/cpp/getting-started.md

```diff
@@ -1,185 +1,393 @@
 ---
 title: Getting Started
+description: Get telemetry for your app in less than 5 minutes!
+cSpell:ignore: oatpp rolldice
 weight: 10
-# prettier-ignore
-cSpell:ignore: Bazel DBUILD devel DWITH helloworld libcurl openssl tracestate xcode
 ---
 
-Welcome to the OpenTelemetry C++ getting started guide! This guide will walk you
-through the basic steps in installing, instrumenting with, configuring, and
-exporting data from OpenTelemetry.
+This page will show you how to get started with OpenTelemetry in C++.
 
-You can use [CMake](https://cmake.org/) or [Bazel](https://bazel.build/) for
-building OpenTelemetry C++. The following getting started guide will make use of
-CMake and only provide you the most essential steps to have a working example
-application (a HTTP server & HTTP client). For more details read
-[these instructions](https://github.com/open-telemetry/opentelemetry-cpp/blob/main/INSTALL.md).
+You will learn how to instrument a simple C++ application automatically,
+such that [traces](/docs/concepts/signals/traces/), [metrics](/docs/concepts/signals/metrics/) and [logs](/docs/concepts/signals/logs/) are emitted to the terminal.
 
-## Prerequisites
+## Prerequisites 
 
-You can build OpenTelemetry C++ on Windows, macOS or Linux. First you need to
-install some dependencies:
+Ensure that you have the following installed locally:
 
-{{< tabpane text=true >}} {{% tab "Linux (apt)" %}}
+- Git
+- C++ compiler supporting C++ version >= 14
+- Make 
+- CMake version >= 3.1
+
+## Example Application 
+
+The following example uses a basic [Oat++](https://oatpp.io/) application. If you are 
+not using Oat++, that's OK - you can use OpenTelemetry C++ with any other web framework as well.
+
+## Setup 
+
+- Create a folder named `otel-cpp-starter`.
+- move into the newly created folder. This will serve as your working directory.
+- After setting up dependencies, your directory structure should resemble this:
+   ```plaintext 
+   otel-cpp-starter
+   │
+   ├── oatpp
+   ├── opentelemetry-cpp
+   └── roll-dice
+
+   ```
+  
+
+## Dependencies 
+
+To begin, install  Oat++ locally using the [source code](https://github.com/oatpp) and `make`, following these steps:
+
+1. Obtain the Oat++ source code by cloning from the [oatpp/oatpp](https://github.com/oatpp/oatpp) GitHub repository.
+
+   ```bash 
+   git clone https://github.com/oatpp/oatpp.git
+   ```
+
+2. Navigate to the `oatpp` directory.
+
+   ```bash 
+   cd oatpp
+   ```
+
+3. Create a `build` subdirectory and navigate into it.
+   ```bash
+   mkdir build
+   cd build
+   ```
+
+4. Build oatpp using the `cmake` and `make` commands. This command will trigger the build process specified in the `CMakeLists.txt` included in the oatpp source code.
+
+   ```bash
+   cmake ..
+   make
+
+5. Install oatpp.
+
+   This command will install the built oatpp library and headers on your system, making it accessible for development in your project. 
+
+   ```bash
+   make install
```

Comment:
```suggestion
   sudo make install
```

On most systems this will require sudo permissions.

Note: We should mention either here or at the end, that people can uninstall it again with `make uninstall` ( I guess)